### PR TITLE
moved headers into credits

### LINKED ISSUE #801: Get rid of headers
I mean, do we really need stuff like the following around?
```
/*******************************************************************\

Module: Format String Parser

Author: CM Wintersteiger

\*******************************************************************/
```

### PR DESCRIPTION
**Pull request by rafaelsamenezes**
Should close #801 

I've removed every header and created the credits file. The exceptions are:

- old-frontend
- message
- bigint, I am not sure what we should do there.

If you know the missing emails, let me know.

### CODE REVIEW

File: CREDITS [hunk 1]
```diff
@@ -0,0 +1,16 @@
+This files contains everyone that has contributed to ESBMC in any way. If you were a contributor and your name is not present, please file an issue so that your name can be added:
+
+- Daniel Kroening, kroening@kroening.com
+- Lucas Cordeiro
```
Comment (lucasccordeiro) [inline CREDITS]: Can you add my email?

lucas.cordeiro@manchester.ac.uk

File: CREDITS [hunk 2]
```diff
@@ -0,0 +1,16 @@
+This files contains everyone that has contributed to ESBMC in any way. If you were a contributor and your name is not present, please file an issue so that your name can be added:
```
Comment (lucasccordeiro) [inline CREDITS:1]: @shmarovfedor is missing here.

Comment (lucasccordeiro) [inline CREDITS:1]: @feliperodri is missing.

File: src/util/symbol_generator.h [hunk 3]
```diff
@@ -1,18 +1,3 @@
-/*******************************************************************\
- Module: Symbol Generator
- Author: Franz Brauße (interface)
-
- Description: This is a simple encapsulation of the contextt::move()
```
Comment (lucasccordeiro) [inline src/util/symbol_generator.h:5]: Shall we keep this description?

File: src/jimple-frontend/AST/jimple_ast.h [hunk 4]
```diff
@@ -1,11 +1,3 @@
-/*******************************************************************\
-Module: Jimple AST Interface
-Author: Rafael Sá Menezes
-Date: September 2021
-Description: This interface will define every method that needs to
```
Comment (lucasccordeiro) [inline src/jimple-frontend/AST/jimple_ast.h:5]: Shall we keep this description?

File: src/jimple-frontend/AST/jimple_class_member.h [hunk 5]
```diff
@@ -1,11 +1,3 @@
-/*******************************************************************\
-Module: Jimple Class Member Interface
-Author: Rafael Sá Menezes
-Date: September 2021
-Description: This interface will hold anything
```
Comment (lucasccordeiro) [inline src/jimple-frontend/AST/jimple_class_member.h:5]: Shall we keep this description?

File: src/jimple-frontend/AST/jimple_declaration.h [hunk 6]
```diff
@@ -1,10 +1,3 @@
-/*******************************************************************\
-Module: Jimple Declaration class
-Author: Rafael Sá Menezes
-Date: September 2021
-Description: This class will hold any declaration statements
```
Comment (lucasccordeiro) [inline src/jimple-frontend/AST/jimple_declaration.h:5]: Shall we keep this description?

File: src/jimple-frontend/AST/jimple_expr.h [hunk 7]
```diff
@@ -1,19 +1,11 @@
-/*******************************************************************\
-Module: Jimple Expr Interface
-Author: Rafael Sá Menezes
-Date: September 2021
-Description: This interface will hold jimple expressions such as
```
Comment (lucasccordeiro) [inline src/jimple-frontend/AST/jimple_expr.h:5]: Shall we keep this description?

File: src/jimple-frontend/AST/jimple_file.h [hunk 8]
```diff
@@ -1,10 +1,3 @@
-/*******************************************************************\
-Module: Jimple File AST
-Author: Rafael Sá Menezes
-Date: September 2021
-Description: Jimple File AST parser and holder
```
Comment (lucasccordeiro) [inline src/jimple-frontend/AST/jimple_file.h:5]: Shall we keep this description?

**Review by lucasccordeiro — CHANGES_REQUESTED** [review]

File: CREDITS (hunk 2 above)
Comment (rafaelsamenezes) [reply CREDITS:1]: added

Comment (rafaelsamenezes) [reply CREDITS:1]: added

**Review by lucasccordeiro — APPROVED** [review]

File: src/jimple-frontend/AST/jimple_ast.h (hunk 4 above)
Comment (mikhailramalho) [reply src/jimple-frontend/AST/jimple_ast.h:5]: I don't think so, it's not like they add a lot.

**Review by mikhailramalho — APPROVED** [review]
LGTM.

Can you change my name to "Mikhail R. Gadelha" and add my personal e-mail?

Adding the esbmc team first would be helpful, instead of mixing with the other contributors.

**Pull request merged**